content/version/2/0/code_of_conduct.pt-br.md

```diff
@@ -0,0 +1,90 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Código de Conduta para Colaboradores
+
+## Nossa promessa
+
+Como participantes, colaboradores e lideranças, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
+
+Comprometemo-nos a agir e interagir de maneiras que contribuem para uma comunidade aberta, acolhedora, diversificada, inclusiva e saudável.
+
+## Nossos padrões
+
+Exemplos de comportamentos que contribuem para criar um ambiente positivo para a nossa comunidade incluem:
+
+* Demonstrar empatia e bondade com as outras pessoas
+* Respeitar opiniões, pontos de vista e experiências contrárias
+* Dar e receber feedbacks construtivos de maneira respeitosa
+* Assumir responsabilidade, pedir desculpas às pessoas afetadas por nossos erros e aprender com a experiência
+* Focar no que é melhor não só para nós individualmente, mas para a comunidade em geral
+
+Exemplos de comportamentos inaceitáveis incluem:
+
+* Uso de linguagem ou imagens sexuais e atenção ou avanço sexual de qualquer natureza
```

Comment:
"Sexualizada" seems to be a better term here to include the "non-explicit content with a sexual intention"
I didn't know what could be "attention and advances" (in Portuguese and in English) but I can imagine

A suggestion to keep the phrase more similar to the English version
```suggestion
* Uso de linguagem ou imagens sexualizadas, atenção sexual ou avanços de qualquer natureza
```

Reply:
I agree with you, I don't think there's a direct and accurate way of translating this. So I'm going for something more commonly used language and now I'm sticking with this:

> Uso de linguagem ou imagens sexualizadas, bem como o assédio sexual ou de qualquer natureza

I think this represents well what the english version means.